packages/ra-core/src/form/useGetFormValues.ts

```diff
@@ -0,0 +1,12 @@
+import { useForm } from 'react-final-form';
+
+/**
+ * This hook returns a function that can be used to get the current values of the form.
+ */
+export const useGetFormValues = <ValuesType = Record<string, any>>() => {
```

Comment:
Could we avoid adding this new indirection? I don't want to hide react-final-form, because we're not adding anything better (for now)

Reply:
Removed but I feel bad about it